addons/map/XEH_postInitClient.sqf

```diff
@@ -19,4 +18,10 @@ call FUNC(determineZoom);
     ((findDisplay 12) displayCtrl 51) ctrlAddEventHandler ["Draw", {[] call FUNC(updateMapEffects);}];
 };
 
-ADDON = true;
+["SettingsInitialized", {
+    // Start Blue Force Tracking if Enabled
+    if (GVAR(BFT_Enabled)) then {
```

Comment:
That XEH won't even execute on Server since it's `clientInit`, but good point on Headless. I added `exitWith` in the beginning as there is nothing that should run on Headless at all in the map module afaik.